src/modules/Builder/utils/getEntityReportFields.test.ts

```diff
@@ -0,0 +1,135 @@
+import { getEntityReportFields, FlowReportFieldsPrepareType } from './getEntityReportFields';
+
+const activities = [
+  {
+    name: 'Activity-1',
+    id: 'activity-id-1',
+    key: 'activity-key-1',
+    items: [{ name: 'Item-1', id: 'id-1', key: 'item-key-1' }],
+  },
+  {
+    name: 'Activity-2',
+    id: 'activity-id-2',
+    key: 'activity-key-2',
+    items: [{ name: 'Item-2', id: 'id-2', key: 'item-key-2' }],
+  },
+];
+
+const activityItems = [
+  { name: 'Item-1', id: 'id-1', key: 'item-key-1' },
+  { name: 'Item-2', id: 'id-2', key: 'item-key-2' },
+];
+
+describe('getEntityReportFields', () => {
+  test('returns empty reportIncludedActivityName for KeyToName type when reportActivity and activities are not provided', () => {
```

Comment:
could you pls also reformat these tests to test.each?

Reply:
too longs lines

Reply:
you could use long descriptions as variables, for example const description1 = '...

Reply:
ok